manager/src/test/resources/application.properties

```diff
@@ -20,6 +20,8 @@ event-bridge.kafka.sasl.mechanism=PLAIN
 event-bridge.rhoas.instance-api.host=http://test-instance-api.redhat.com
 event-bridge.rhoas.mgmt-api.host=http://test-management-api.redhat.com
 
-# Workers require scheduling however we only enable it for the applicable tests.
-# See Quarkus Test Profile WorkerSchedulerProfile
-quarkus.scheduler.enabled=false
+# Quartz Scheduler configuration for tests
+event-bridge.resources.workers.frequency-seconds=1
+quarkus.quartz.start-mode=forced
+quarkus.quartz.clustered=false
+quarkus.quartz.store-type=ram
```

Comment:
@manstis Is there any reason why we can't or shouldn't use the JDBC store in our tests?

Reply:
Speed... otherwise no reason at all.